### PR TITLE
Add options to configure webhook in helm chart and improve the relevant doc.

### PR DESCRIPTION
**Pull request by dashanji**
What do these changes do?
-------------------------

* Add options to configure webhook.
* Improve the doc.
* Delete the useless script.

Related issue number
--------------------

Related to https://github.com/v6d-io/v6d/issues/1059

### CODE REVIEW

File: charts/test.yaml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: mysecret
+type: Opaque
+data:
+  USER_NAME: YWRtaW4=
+  PASSWORD: MWYyZDFlMmU2N2Rm
```
Comment (sighingnow) [inline charts/test.yaml:8]: Why secret is needed?

File: charts/vineyard-operator/README.md [hunk 2]
```diff
@@ -6,39 +6,94 @@ A helm chart for [vineyard operator][3], which manages all relavant components a
...
+  image: ghcr.io/v6d-io/v6d/vineyardd:alpine-latest
+  replicas: 2
+  imagePullPolicy: IfNotPresent
+  syncCRDs: true
+  enableMetrics: false
+  etcd:
+    replicas: 3
+  service:
+    type: ClusterIP
+    port: 9600
```
Comment (sighingnow) [inline charts/vineyard-operator/README.md:69]: Omit default value please.

Please refer to the revised example in `vineyard-operator.rst`.

File: charts/vineyard-operator/README.md [hunk 3]
```diff
@@ -6,39 +6,94 @@ A helm chart for [vineyard operator][3], which manages all relavant components a
...
+EOF
+
```
Comment (sighingnow) [inline charts/vineyard-operator/README.md:71]: No this empty line.

File: charts/vineyard-operator/README.md [hunk 4]
```diff
@@ -6,39 +6,94 @@ A helm chart for [vineyard operator][3], which manages all relavant components a
...
 
+$ helm uninstall vineyard-operator
+
+```
 More information about the helm chart could be found at [artifacthub][1] and [parameters][2].
 
 ## Values
 
-The following table lists the configurable parameters of the Vineyard Operator chart and their default values.
+The following table lists the configurable parameters of the Vineyard Operator chart and their default values. Besides, you can refer the [doc](https://v6d.io/notes/vineyard-operator.html) to get more detail about the vineyard operator.
```
Comment (sighingnow) [inline charts/vineyard-operator/README.md:87]: Don't make lines so wide.

**Review by sighingnow — APPROVED** [review]
Could you please have some briefing about the changes in PR title?

Thanks!

File: charts/test.yaml (hunk 1 above)
Comment (dashanji) [reply charts/test.yaml:8]: sorry, it's only for local tests.

**Comment by dashanji** [comment]
Okay, all comments done, please take another look at this.

**Pull request merged**